public void test_execute_3() throws Exception {
  Connection conn=dataSource.getConnection();
  PreparedStatement stmt=conn.prepareStatement("SELELCT 1");
  stmt.execute();
  ((DruidPooledStatement)stmt).getStatement().close();
{
    SQLException error=null;
    try {
      stmt.execute();
    }
 catch (    SQLException ex) {
      error=ex;
    }
    Assert.assertNotNull(error);
  }
  stmt.close();
  conn.close();
}
